### PR TITLE
Storage sample docs #6

### LINKED ISSUE #6: Update storage examples
/kind feature

**Describe the solution you'd like**
update api group and kfserving name reference 
https://github.com/kubeflow/kfserving/tree/master/docs/samples/storage

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

### PR DESCRIPTION
**Pull request by Suresh-Nakkeran**
fixes #6

### CODE REVIEW

File: docs/modelserving/storage/s3/mnist-s3.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: "serving.kserve.io/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: "mnist-s3"
+spec:
+  predictor:
+    serviceAccountName: sa
+    tensorflow:
+      storageUri: "s3://kserve-examples/mnist"
```
Comment (yuzisun) [inline docs/modelserving/storage/s3/mnist-s3.yaml:9]: Does this s3 bucket exist?

File: docs/modelserving/storage/azure/azure.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+
+# Predict on a InferenceService with saved model on Azure
+
+## Using Public Azure Blobs
```
Comment (yuzisun) [inline docs/modelserving/storage/azure/azure.md:4]: This example is likely outdated, let's rework on it and merge the PR https://github.com/kubeflow/kfserving/pull/1750 onto KServe for the azure blob client upgrade.

File: docs/modelserving/storage/s3/mnist-s3.yaml (hunk 1 above)
Comment (Suresh-Nakkeran) [reply docs/modelserving/storage/s3/mnist-s3.yaml:9]: @yuzisun we have this s3 bucket but it is private.

File: docs/modelserving/storage/s3/s3.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
+# Predict on a InferenceService with a saved model on S3
+
+## Setup
+
+1. Your ~/.kube/config should point to a cluster with [KServe installed](../../../admin/serverless.md).
+2. Your cluster's Istio Ingress gateway must be [network accessible](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/).
+3. Your cluster's Istio Egresss gateway must [allow accessing S3 Storage](https://knative.dev/docs/serving/outbound-network-access/)
```
[the remark this answers is not in the document]
Comment (Suresh-Nakkeran) [reply docs/modelserving/storage/s3/s3.md]: @yuzisun removed.

File: docs/modelserving/storage/pvc/pvc.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+kubectl apply -f pv-model-store.yaml
+
+kubectl exec -it model-store-pod -- bash
+```
+
+In different terminal, copy the model from local into PV.
+
+=== "kubectl"
+```bash
+kubectl cp model.jcllib model-store-pod:/pv/model.jcllib -c model-store
```
Comment (yuzisun) [inline docs/modelserving/storage/pvc/pvc.md:60]: jcllib -> joblib?

File: docs/modelserving/storage/pvc/pvc.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+=== "yaml"
+```yaml
+apiVersion: "serving.kserve.io/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: "sklearn-pvc"
+spec:
+  predictor:
+    sklearn:
+      storageUri: "pvc://${PVC_NAME}/model.jcllib"
```
Comment (yuzisun) [inline docs/modelserving/storage/pvc/pvc.md:76]: jcllib -> joblib?

**Comment by yuzisun** [comment]
@Suresh-Nakkeran Can you help rebase from main branch to fix the netlify CI build issue?

File: docs/modelserving/storage/pvc/pvc.md (hunk 5 above)
Comment (Suresh-Nakkeran) [reply docs/modelserving/storage/pvc/pvc.md]: @yuzisun typo error! I updated.

File: docs/modelserving/storage/pvc/pvc.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+kubectl apply -f pv-and-pvc.yaml
+```
+
+### Copy model to PV
+
+Run pod `model-store-pod` and login into container `model-store`.
+
+=== "kubectl"
+```bash
+kubectl apply -f pv-model-store.yaml
```
Comment (yuzisun) [inline docs/modelserving/storage/pvc/pvc.md:51]: Can we show the content of `pv-model-store.yaml`?

File: docs/modelserving/storage/pvc/pvc.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+```
+
+In different terminal, copy the model from local into PV.
+
+=== "kubectl"
+```bash
+kubectl cp model.joblib model-store-pod:/pv/model.joblib -c model-store
+```
+
+## Specify and create the `InferenceService`
```
Comment (yuzisun) [inline docs/modelserving/storage/pvc/pvc.md:63]:
```suggestion
## Deploy `InferenceService` with models on PVC
```

File: docs/modelserving/storage/pvc/pvc.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+    sklearn:
+      storageUri: "pvc://${PVC_NAME}/model.joblib"
+```
+
+=== "kubectl"
+```bash
+kubectl apply -f sklearn-pvc.yaml
+```
+
+## Check the InferenceService
```
Comment (yuzisun) [inline docs/modelserving/storage/pvc/pvc.md:84]: Can you help add an additional step to run the actual inference?

**Pull request merged**